flycheck.el

```diff
@@ -4297,8 +4297,10 @@ POS defaults to `point'."
 
 (defun flycheck-jump-to-error (error)
   "Go to the location of ERROR."
-  (let ((error-copy (copy-flycheck-error error))
-        (buffer (find-file-noselect (flycheck-error-filename error))))
+  (let* ((error-copy (copy-flycheck-error error))
+         (filename (flycheck-error-filename error))
+         (buffer (or (and filename (find-file-noselect filename))
+                     (flycheck-error-buffer error))))
```

Comment:
Shouldn't we use -when-let? The buffer could have been deleted

Reply:
Good idea.